### ISSUE TITLE
Constant cpu usage when running Flowblade.

### ISSUE THREAD

**Issue by u3erprutser** [jliljebl/flowblade-forum#96]
I want to try out flowblade, and so far I installed version 2.2 on a old 32bit laptop and the latest version using virtualbox (linux mint 20) Before ever doing some editing, just starting the program, I noticed high cpu usage on both installations. Is this normal behaviour or can I expect better on freshly installed real hardware? (e8400) 
If it's normal to eat up 30% of the cpu cycles, I don't want to go though the trouble. But if there is a simple fix I'm very interested in the program

**Comment by jliljebl** [comment]
Yes, currently Flowblade produces frames constantly on selected Project profile fps once started, whether or not playback is on. We are just working to introduce new display consumer which may address this Issue.
